app/controllers/requests_controller.rb

```diff
@@ -79,17 +79,17 @@ def destroy
 
   def recap
     @request = Request.find(params[:id])
+    return unless @request
+
     @vm = VSphere::VirtualMachine.find_by_name(@request.name)
     @request.assign_sudo_users(request_params[:sudo_user_ids][1..-1])
```

Comment:
why do we assign the sudo users here. I think thsi should be done in theb change state, since this is where the rest of the request is updated